source/agora/common/crypto/Schnorr.d

```diff
@@ -209,6 +212,25 @@ public bool verify (T) (const ref Point X, const ref Signature s, auto ref T dat
     return S == RcX;
 }
 
+/// Publickey(Ed25519) to Point(Curve25519)
+public static Point pkToCurvePoint (PublicKey pk) nothrow @nogc
```

Comment:
```suggestion
public Point pkToCurvePoint (PublicKey pk) nothrow @nogc
```

`static` is unnecessary as this function is at module level (unlike in C)

Reply:
Applied the suggest.